src/commands/ml-models-create.ts

```diff
@@ -0,0 +1,22 @@
+import { Command } from "../command";
+import { FirebaseModel } from "../ml/models";
+
+module.exports = new Command("ml:models:create <displayName> [source]")
```

Comment:
```suggestion
export default new Command("ml:models:create <displayName> [source]")
```

`src/command.ts` is wired to expect default exports from TS files

Reply:
Driving by ... is this true?  Many commands don't do this:
https://github.com/firebase/firebase-tools/blob/master/src/commands/emulators-start.ts#L6

Reply:
done

Reply:
I believe the majority do though. Functionally it's the same.